agent/consul/intention_endpoint.go

```diff
@@ -134,12 +146,12 @@ func (s *Intention) prepareApplyUpdate(authz acl.Authorizer, entMeta *structs.En
 
 // prepareApplyDelete ensures that the intention specified by the ID in the request exists
 // and that the requester is authorized to delete it
-func (s *Intention) prepareApplyDelete(authz acl.Authorizer, entMeta *structs.EnterpriseMeta, args *structs.IntentionRequest) error {
+func (s *Intention) prepareApplyDelete(ident structs.ACLIdentity, authz acl.Authorizer, entMeta *structs.EnterpriseMeta, args *structs.IntentionRequest) error {
 	// If this is not a create, then we have to verify the ID.
 	state := s.srv.fsm.State()
 	_, ixn, err := state.IntentionGet(nil, args.Intention.ID)
 	if err != nil {
-		return fmt.Errorf("Intention lookup failed: %v", err)
+		return fmt.Errorf("Intention lookup failed: %q", err)
```

Comment:
Why `%q` vs `%v`? I think the only difference in this case will be that the error is quoted. Is that desired?

Reply:
No deeper point here, I was chasing down a stack overflow bug caused by the mis-implemented interface we discussed before, and read that %+v can sometimes cause a loop if it resolves to a call to `.String()` naturally I removed a whole bunch of %v anything in paranoia before figuring out the actual bug 😅 

Will revert in a revision tomorrow